src/App.js

```diff
@@ -19,6 +25,10 @@ const App = () => {
                <Route path="/login" element={<LoginPage />} />
                <Route path="/signup" element={<SignUpPage />} />
             </Route>
+            //private routes
```

Comment:
suggestion: this should be inside braces `{}`

Reply:
Done.